test/SendRecv_OutOfPlace.cpp

```diff
@@ -0,0 +1,90 @@
+/*************************************************************************
+ * Copyright (c) 2022 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+#include "TestBed.hpp"
+
+namespace RcclUnitTesting
+{
+  TEST(SendRecv, OutOfPlace)
+  {
+    TestBed testBed;
+
+    // Configuration
+    std::vector<ncclFunc_t>     const  funcType        = {ncclCollSend, ncclCollRecv}; // akollias

```

Comment:
This is rather misleading because of how it's used compared to other tests.
For this test, it would be more clear if you use ncclCollSend, ncclCollRecv directly in the SetCollectivesArgs call.